### PR TITLE
Fix for #10

### PR DESCRIPTION
**Pull request by RobertPeek**
Addition of the this.mediaCanAutoplay variable so that it only autoplays once.

Slight rework of inview so that it only looks for the element being inview on the Top or Both.

### CODE REVIEW

**Comment by dancgray** [comment]
I think we should have the option to set autoplay once (true/false) in the JSON.

File: js/adapt-media-autoplay.js [hunk 1]
```diff
@@ -197,19 +199,15 @@ define(function(require) {
             }
         },
 
-        inview: function(event, visible, visiblePartX, visiblePartY) {
+        inview: function(event, visible, visiblePartX, visiblePartY) {
```
Comment (dancgray) [inline js/adapt-media-autoplay.js]: Can we get rid of trailing whitespace :)

**Comment by RobertPeek** [comment]
Added that feature above.

**Comment by dancgray** [comment]
+1

**Pull request merged**